Doc/library/asyncio-dev.rst

```diff
@@ -73,8 +73,8 @@ event loop, no other Tasks can run in the same thread.  When a Task
 executes an ``await`` expression, the running Task gets suspended, and
 the event loop executes the next Task.
 
-To schedule a callback from a different OS thread, the
-:meth:`loop.call_soon_threadsafe` method should be used. Example::
+To schedule a :ref:`callback <callback>` from a different OS thread,
```

Comment:
The [documentation](https://docs.python.org/3/library/asyncio-task.html#asyncio.create_task) for `asyncio.create_task()` says "Wrap the *coro* coroutine" instead of "Wrap a *coro* coroutine."